hostinit/setup.go

```diff
@@ -366,13 +366,23 @@ func (init *HostInit) IsHostReady(host *host.Host) (bool, error) {
 	if err != nil {
 		return false, errors.Wrapf(err, "failed to get cloud host for %s", host.Id)
 	}
-	reachable, err := cloudHost.IsSSHReachable()
-	if err != nil {
-		return false, errors.Wrapf(err, "error checking if host %s is reachable", host.Id)
+	reachableTimes := 0
+	for {
+		reachable, err := cloudHost.IsSSHReachable()
+		if err != nil {
+			return false, errors.Wrapf(err, "error checking if host %s is reachable", host.Id)
+		}
+		if !reachable {
+			return reachable, nil
+		}
+		if reachableTimes < 3 {
+			reachableTimes++
+			time.Sleep(time.Second)
+			continue
+		}
+		time.Sleep(time.Second)
```

Comment:
I'd omit this.

Reply:
I simplified the loop a little at Richard's request. The reason to leave a sleep at the end after a successful reachability check is that the current, broken behavior includes SCP-ing nearly the same instant that we succeed at reachability.